Sources/IosAnalyticsDebugger/Resources/BarDebugger.xib

```diff
@@ -94,8 +95,9 @@
                 <constraint firstItem="hmy-Gf-lbs" firstAttribute="leading" secondItem="vUN-kp-3ea" secondAttribute="leading" id="3k1-zj-n8k"/>
                 <constraint firstItem="8pX-Qi-zk8" firstAttribute="leading" secondItem="vUN-kp-3ea" secondAttribute="leading" id="Oa0-Ti-vCR"/>
                 <constraint firstItem="vUN-kp-3ea" firstAttribute="trailing" secondItem="hmy-Gf-lbs" secondAttribute="trailing" id="OeS-jd-71c"/>
-                <constraint firstItem="vUN-kp-3ea" firstAttribute="bottom" secondItem="hmy-Gf-lbs" secondAttribute="bottom" id="XN1-PQ-Blv"/>
```

Comment:
Don't constrain the bar debugger's subviews to the safe area. Instead, the entire bar itself is kept out of the safe area by the drag calculations. Previously, the view's visible bar UI didn't match the main view making the tap and drag gestures hard to activate.